Website and Playground Setup and Run Instructions (#5312)
prettier/prettier: website/README.md

# Prettier Website

https://prettier.io/

## Setup

> 1. Optionally, download the Playground parser libs (to `prettier/website/static/lib/`) by running the following in the project root dir:

Reviewer: ```suggestion
1. Build the Prettier browser libs for playground by running the following in the project root dir:
```

       cd your/path/to/prettier
       yarn build-docs

>     (To build for master, use `yarn build-docs PULL_REQUEST=true`)

Reviewer 2: Should this be `PULL_REQUEST=true yarn build-docs`?
